operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [ENHANCEMENT] Rollout-operator: upgraded to v0.10.1. #7125
 * [ENHANCEMENT] Query-frontend: configured `-shutdown-delay`, `-server.grpc.keepalive.max-connection-age` and termination grace period to reduce the likelihood of queries hitting terminated query-frontends. #7129
 * [BUGFIX] Metamonitoring: update dashboards to drop unsupported `step` parameter in targets. #7157
+* [ENHANCEMENT] Add the possibility to create a dedicated serviceAccount for the `ruler` component.
```

Comment:
can you move this one line up, add the number of the PR at the end, and also add the actual value that needs to be set `ruler.serivceAcount.create`?

Reply:
Sure.